src-v2/Not/Not.Blazor/TM/NotChildren.razor

```diff
@@ -0,0 +1,62 @@
+﻿@using Not.Blazor.Components
+@using Not.Blazor.Forms
+@using Not.Blazor.Mud.Components
+@using Not.Blazor.Navigation
+@using Not.Blazor.TM.Forms.Components
+@using Not.Services
+
+@typeparam T where T : DomainEntity
+@typeparam TForm where TForm : NotForm<T>
+
+<MudPaper Elevation="0" Class="children-component half-width">
+@if (_parentBehind.Children.Any())
+{
+    <NotH2 Class="pt-6 pl-6">@Name</NotH2>
+    <NotButtonCreate Class="mt-1 ml-6 mb-3" OnClick="_contestantNavigator.Create" />
+    <MudDivider />
+    @foreach (var item in Items)
+    {
+        <p class="mt-1 ml-6 mb-1">
+            @item.ToString() 
```

Comment:
Think `ToString()` will be called implicitly by the blazor framework. Try it and remove it if isn't necessary.